chapter3.Rmd

```diff
@@ -597,7 +597,7 @@ success_msg("Great! Continue to the next exercise.")
 --- type:NormalExercise lang:r xp:100 skills:1
 ## Create submatrices
 
-Last but not least, there's the selection of entire matrices from bigger matrices. As soon as a vector is not sufficient to store the information you're selecting, you're ending up with a new matrix. If you want to a submatrix that comprises rows 1 and 4 and columns 2 and 3 of a matrix `m`, to following call will help you out:
+Last but not least, you can create submatrices from larger matrices. If a vector is not sufficient to store the information you want to select, you need to create a new matrix. If you want to a submatrix that comprises rows 1 and 4 and columns 2 and 3 of a matrix `m`, to following call will help you out:
```

Comment:
Agree this is better, but I think you still need an extra word here: "If you want to a submatrix"